gatling-tests/run-tests.ps1

```
@@ -1,11 +1,10 @@
 #!/usr/bin/env pwsh
 
 param (
-    [string]$TestProfile = "default",
+    [string]$TestProfile = "standard",
     [string]$BaseUrl = "http://localhost:8765",
-    [int]$Users = 10,
-    [int]$Duration = 60,
-    [int]$KAFKA_INIT_DELAY=15
+    [int]$Users = 50,
+    [int]$Duration = 300,
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'Duration' has been declared but not used.

(PSReviewUnusedParameter)